docs/source/installation.md

```diff
@@ -102,22 +102,34 @@ By default, the installation steps will only download and install the minimal re
 Optional dependencies can be installed using [the extras syntax](https://packaging.python.org/tutorials/installing-packages/#installing-setuptools-extras) to support additional features.
 
 For example, to install MONAI with Nibabel and Scikit-image support:
-```
+```bash
 git clone https://github.com/Project-MONAI/MONAI.git
 cd MONAI/
 pip install -e '.[nibabel,skimage]'
 ```
 
 Alternatively, to install all optional dependencies:
-```
+```bash
 git clone https://github.com/Project-MONAI/MONAI.git
 cd MONAI/
 pip install -e '.[all]'
 ```
 
 To install all optional dependencies for MONAI development:
-```
+```bash
 git clone https://github.com/Project-MONAI/MONAI.git
 cd MONAI/
 pip install -r requirements-dev.txt
 ```
+
+Since MONAI v0.2.0, the extras syntax such as `pip install 'monai[nibabel]'` is available via PyPI.
+
+- The options are
+```
+[nibabel, skimage, pillow, tensorboard, ignite]
```

Comment:
missing `all` option in the list?
And could you please add what's the default behavior of "pip install monai"?

Thanks.

Reply:
the option 'all' is explained at line 133, and the beginning of this file explains the min. requirements

Reply:
Hi @wyli ,

1. I mean should `all` in the [... ...] list?
2. For the "pip install monai", I feel it's better to add "without optional features":
To install the current milestone release **without optional features**:
```
pip install monai
```
Otherwise, people may complain why missing XXX after installation.

Thanks.

Reply:
Hi Nic, for 1: ‘all’ is different from the options in the  list, the options could be combined: pip install ‘monai[nibabel,skimage]’, but combining ‘all’ with the others such as pip install ’monai[all,skimage]’ is unnecessary. for 2, I think that’s a valid concern, this should be addressed by improving the relevant error messages/system info when missing the optional package. I’ll do it in a separate PR soon. Depends on the feedback, after v0.2.0 the default behaviour could be changed to: install all optional unless user specifies installing monai-core only.

Reply:
As we don't have a clear error messages for missing optional packages now, why not keep the default behavior of "pip install monai" as v0.1 to install all the things?
If users don't want to install all things, they can use our new optional mechanism.
We can change the default to only install core modules when we have clear messages.
What do you think?
Hi @ericspod , could you please also help share some comments?

Thanks.

Reply:
I think we should wait for more feedback instead of creating imagery issues. also it should be discussed in a separate ticket. this PR is only doing minor documentation update.